JSDemos/Demos/TagBox/GroupedItems/Vue/Group.vue

```diff
@@ -4,15 +4,12 @@
     {{ itemData.key }}
   </div>
 </template>
-<script>
-export default {
-  props: {
-    itemData: {
-      type: Object,
-      default: () => {},
-    },
-  },
-};
+<script setup lang="ts">
+withDefaults(defineProps<{
+  itemData?: object
```

Comment:
in VSCode I see error "Parsing error: Unexpected token ? eslint".
Can we check all this errors for all changes?

Reply:
i fixed it by update eslint